src/cephadm/cephadm.py

```diff
@@ -938,6 +938,64 @@ def get_tcmu_runner_container(self):
 ##################################
 
 
+class CephExporter(object):
```

Comment:
would it make sense to put it into a new file src/cephadm/exporter.py ?

Reply:
probably, but we had decided we were going to wait until after the initial reef release to actually split up the binary at all so the renaming `cephadm.py -> cephadm` workaround is still available if there turns out to be any issues with the compile cephadm stuff.